### ISSUE TITLE
Add pipeline

### ISSUE THREAD

**Issue by pavelsvagr** [AckeeCZ/configuru-extension#2]
Add pipeline that will publish the new version of the extension to Visual Studio Marketplace once merged to master

**Comment by smoliji** [comment]
Or keep the publish on manual release? I believe this is similar to what we do with NPM package publishing. I have this set up on healthz and I think we agreed it is a good approach: https://github.com/AckeeCZ/node-healthz/blob/master/.github/workflows/npm-publish.yml#L2-L4

**Comment by pavelsvagr** [comment]
Yes, sorry, this was a wrongly worded issue. We want to keep the manual deploy.
